### PR TITLE
console log

### PR DESCRIPTION
**Pull request by liorbond**
https://fhenix.monday.com/boards/1216577959/views/4451803/pulses/1392626038

### CODE REVIEW

File: contracts/FheOS.sol [hunk 1]
```diff
@@ -1,13 +1,14 @@
 // SPDX-License-Identifier: BSD-3-Clause-Clear
 // solhint-disable one-contract-per-file
-pragma solidity >=0.8.19 <0.8.25;
```
Comment (Cashmaney) [inline contracts/FheOS.sol:3]: Why change this?

File: contracts/Console.sol [hunk 2]
```diff
@@ -0,0 +1,687 @@
+// SPDX-License-Identifier: MIT
```
Comment (Cashmaney) [inline contracts/Console.sol:1]: LGTM - can you please move this to a different folder? i.e. something like /utils/debug/console.log?

File: genConsole.ts [hunk 3]
```diff
@@ -0,0 +1,173 @@
+import { writeFileSync } from "fs";
```
Comment (Cashmaney) [inline genConsole.ts:1]: Also move this to a different folder and add a generate command so we can regenerate this if some changes are needed

**Comment by Cashmaney** [comment]
Add the file to the package.json in `contracts/` otherwise it won't get distributed with a new release

**Review by Cashmaney — CHANGES_REQUESTED** [review]

File: contracts/FheOS.sol (hunk 1 above)
Comment (liorbond) [reply contracts/FheOS.sol:3]: Auto gen, copied from Fheos

**Pull request merged**